src/GeneticSharp.Domain.UnitTests/Populations/PopulationTest.cs

```diff
@@ -66,7 +66,14 @@ public void CreateInitialGeneration_AdamChromosomeCreateNewNull_Exception()
 
             Assert.Catch<InvalidOperationException>(() =>
             {
-                population.CreateInitialGeneration();
+                try
```

Comment:
This can be reverted now. It was modified in the first version due to the fact that exception raised in Parallel.For are wrapped inside an AggregateException. Now with TplPopulation is not required anymore.